iceoryx_utils/include/iceoryx_utils/internal/cxx/expected.inl

```diff
@@ -327,58 +327,6 @@ inline expected<ErrorType>::expected(error<ErrorType>&& errorValue) noexcept
 {
 }
 
-template <typename ErrorType>
```

Comment:
Removed by error?

Reply:
This is special handling for Win/MSVC. Added code and `#if WIN32` to fix it.